internal/manager/run.go

```diff
@@ -64,15 +65,30 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic) e
 	}
 
 	setupLog.Info("getting the kong admin api client configuration")
-	kongConfig, err := setupKongConfig(ctx, setupLog, c)
+	adminClient, err := c.GetKongClient(ctx)
 	if err != nil {
-		return fmt.Errorf("unable to build the kong admin api configuration: %w", err)
-	}
+		return fmt.Errorf("unable to build kong api client: %w", err)
+	}
+
+	var kongRoot map[string]interface{}
+	err = retry.Do(
+		func() error {
+			kongRoot, err = adminClient.Root(ctx)
+			return err
+		},
+		retry.Attempts(c.KongAdminAPIRetries),
+		retry.DelayType(retry.FixedDelay),
```

Comment:
We could add a line for a Retry Delay interval and that could be configurable too. When `retry.Delay()` is not specified, the retry delay interval is not specified, it defaults 100ms between each retry attempt

Reply:
I'm not seeing an immediate need for a retry delay, let's hold on this for now it's trivial to add later if someone comes up with a specific need.